jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -469,6 +469,9 @@ frontend http-in
     http-response add-header <%= rsp_header.gsub(/(?!:\\)( )/, '\ ') %> "<%= value.to_s.gsub(/(?!:\\) /, '\ ') %>"
     <%- end -%>
   <%- end -%>
+  <%- if_p("ha_proxy.true_client_ip_header") do |header| -%>
+      http-request set-header <%= header %> %[src]
```

Comment:
maybe remove two spaces in front of http-request so that this aligns with the rest of options when rendered?